src/tests/interpolation/test_interpolation_finite_element.cc

```diff
@@ -43,28 +43,38 @@ CASE("test_interpolation_finite_element") {
 
     auto func = [](double x) -> double { return std::sin(x * M_PI / 180.); };
 
-    Interpolation interpolation(option::type("finite-element"), fs, pointcloud);
+    util::Config config = util::Config("type", "finite-element").set("max_fraction_elems_to_try", 0.4);
+    Interpolation interpolation(config, fs, pointcloud);
```

Comment:
You can use e.g.
```c++
Interpolation interpolation{option::type("finite-element")|util::Config("max_fraction_elems_to_try",0.4), fs, pointcloud};
```
You can concatenate configurations with the `|` symbol and `option::type("finite-element")` is a shorthand for `util::Config("type","finite-element")`

Reply:
Sounds fine by me, I have made the suggested change in the latest commit.